### PR TITLE
Vulcanexus Galactic Gamble Documentation

### PR DESCRIPTION
**Pull request by EduPonz**
This PR adds documentation for Galactic Gamble

### CODE REVIEW

File: docs/rst/figures/intro/vulcanexus_packages_table.svg
Comment (JLBuenoLopez-eProsima) [inline docs/rst/figures/intro/vulcanexus_packages_table.svg:1]: The image says `Statiscs Backend` instead of `Statistics Backend`.

File: docs/index.rst
Comment (JLBuenoLopez-eProsima) [inline docs/index.rst:40]: This is not true right now. Either adding a note stating that the documentation is still under work or I suggest removing this sentence.

Another option is modifying to state what the documentation really details:

```suggestion
The following documentation describes how to install *Vulcanexus* packages, some tutorials to know how to get started, the supported platforms and releases.
```

**Review by JLBuenoLopez-eProsima — CHANGES_REQUESTED** [review]
I would have added this commit (26e9680e764e2514a94577d7c7d2780c8ebb8611) in a different PR. This branch was supposed to be an intermediary branch before merging all the documentation into the `galactic` branch once all unknowns were solved. This is the reason where I would not approve this PR yet because there is still work to do (mark with `TODO`).

Currently the following information is unknown:

* [x] Vulcanexus repository final URL
* [x] Vulcanexux webpage final URL
* [ ] Vulcanexus Galactic Release Date

**Comment by EduPonz** [comment]
> I would have added this commit ([26e9680](https://github.com/eProsima/vulcanexus/commit/26e9680e764e2514a94577d7c7d2780c8ebb8611)) in a different PR. This branch was supposed to be an intermediary branch before merging all the documentation into the `galactic` branch once all unknowns were solved.

You are completely right! I have removed my commit from here and submitted #15 instead. I've applied your suggestions there.

**Review by JLBuenoLopez-eProsima — APPROVED** [review]
:+1:

**Pull request merged**
